BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/Views/SkontoProceedView.swift

```diff
@@ -165,7 +184,7 @@ class SkontoProceedView: UIView {
 
 private extension SkontoProceedView {
     enum Constants {
-        static let padding: CGFloat = 24
+        static let padding: CGFloat = 16
         static let verticalPadding: CGFloat = 16
```

Comment:
[clarification] I the figma file the top padding of the button is 20. Why did we not use that dimension?

Reply:
In previous version it was 16, haven't doublechecked it, will change. Thanks!